src/commands/exec.yml

```diff
@@ -15,4 +19,6 @@ parameters:
 steps:
   - install:
       version: << parameters.version >>
-  - run: secrethub run << parameters.flags >> --no-prompt -- $SHELL -c '<< parameters.command >>'
+  - run:
+      name: << parameters.command-name >>
```

Comment:
Is setting to the empty string (the default for `command-name`) equivalent to not setting `name`? i.e. does the CircleCI default kick in if no `command-name` is provided via the SecretHub orb?

Reply:
Yes, I tested this.